mos-platform/nes-unrom-512/bank.h

```diff
@@ -0,0 +1,132 @@
+// Copyright 2023 LLVM-MOS Project
+// Licensed under the Apache License, Version 2.0 with LLVM Exceptions.
+// See https://github.com/llvm-mos/llvm-mos-sdk/blob/main/LICENSE for license
+// information.
+
+#ifndef _BANK_H_
+#define _BANK_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Define this in a .c file to use 2-screen horizontal mirroring.
+ */
+#define UNROM_512_USE_2_SCREEN_HORIZONTAL \
+	asm(".globl __mirroring\n.globl __four_screen\n__mirroring = 0\n__four_screen = 0\n")
+
+/**
+ * @brief Define this in a .c file to use 2-screen vertical mirroring.
+ */
+#define UNROM_512_USE_2_SCREEN_VERTICAL \
```

Comment:
Also, man this is a really nice pattern; we should have these for most settings in the SDK I think. NAR here; I made #193 for this.